src/main/java/com/example/animalshelter/controller/AnimalDeletedResponse.java

```diff
@@ -0,0 +1,10 @@
+package com.example.animalshelter.controller;
+
+import com.example.animalshelter.model.Animal;
+import lombok.Data;
+
+@Data
+public class AnimalDeletedResponse {
+
+    private final Animal deletedAnimal;
```

Comment:
Same here, although not sure if this response is required for deleting. I think status would be sufficient enough.
So void in controller method as no response body.

Reply:
Changed to void.